### PR TITLE
Tanmay/email verification

### LINKED ISSUE #51: Error pop-up when registration from an already registered ID

### LINKED ISSUE #60: Sign up: verify email

### PR DESCRIPTION
**Pull request by tnmyk**
Closes #60 
Closes #51

### CODE REVIEW

**Comment by tnmyk** [comment]
Closes #51

File: src/App.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ function App() {
   const history = useHistory();
   return (
     <div className="App">
-      <AuthProvider>
-        <Router history={history}>
+      <Router history={history}>
+        <AuthProvider>
           <Navbar />
           <Switch>
```
Comment (tnmyk) [inline src/App.js:30]: sadsadasdwdsad

**Review by nitashadhingra — CHANGES_REQUESTED** [review]
Login page: DialogueContent:
Email not verified
    Please check your inbox and verify your email to sign in.

Login page:
before verification check, also check if user used that id to register in the first place or not

**Review by nitashadhingra — CHANGES_REQUESTED** [review]
On signup page, after clicking on AGREE on guidelines, there's no popup or navigation of page 
Popup text should be : "Please check your inbox and verify your email"

**Review by nitashadhingra — CHANGES_REQUESTED** [review]
1. Can u change all the window.alerts to Dialogs (No title needed)
2. Incorrect password check should also be added!! (flow of checks: registered? -> verified? -> correct password)

**Pull request merged**